addons/binding/org.openhab.binding.sleepiq/src/main/java/org/openhab/binding/sleepiq/config/SleepIQCloudConfiguration.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.sleepiq.config;
+
+/**
+ * Configuration class for the SleepIQ cloud.
+ *
+ * @author Gregory Moyer - Initial contribution
+ */
+public class SleepIQCloudConfiguration {
+
+    public static final String USERNAME = "username";
+    public static final String PASSWORD = "password";
+    public static final String POLLING_INTERVAL = "pollingInterval";
+    public static final String LOGGING = "logging";
```

Comment:
Listening for log level change is non-trivial. You'r right that my solution does only allow a log level change after re-initialization 👍 

Thanks for the complete explanation, it's not pretty but its very hard to do better.